scripts/gene_relationships.py

```diff
@@ -200,14 +206,43 @@ def get_relationship_in_expression_space(
                 corr_subset = corr_subset.drop(columns=co_operonic_genes_to_remove)
 
         offset_max = corr_subset.shape[1]
-
         # Note: we are removing genes that are co-operonic first
         # and then finding the most co-expressed genes
-        top_corr_genes = list(corr_subset.loc[gene].nlargest(offset_max).index[1:])
+
+        # If calculating the confidence interval, there will be
+        # duplicate rows due to sampling so we need to select
+        # one of the rows
```

Comment:
`.duplicated()` with no arguments will take the first row by default, right? Might be good to mention here, if it could potentially be important.

Reply:
Yes I'll definitely add this to the comment as this caused a bit of an issue for me when I was coding because I was expecting the function to drop duplicates which is not what its doing.